docs.md

```diff
@@ -75,11 +76,42 @@ export default compose(
 
 Returns **[Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)** A HOC that can be used to wrap a component.
 
+## SUPPORTED_REACT_VERSION
```

Comment:
🤔 🤔 🤔 

Haha you'll probably have to use the @name property in the doc comment.

Reply:
whoa, weird!